public/i18n/en.yaml

```diff
@@ -962,7 +962,7 @@ modules:
       module-already-exists: <0> {{moduleTemplate}} </0> already exists in the <0> {{namespace}} </0> namespace.
       module-templates-to-add: ModuleTemplates to add
       modules-wont-be-added: Following ModuleTemplates already exist in the cluster and cannot be added.
-      namespace-description: Namespace where ModuleTemplates are applied. If left empty, the namespace defined in the ModuleTemplates is used.
+      namespace-description: Namespace where ModuleTemplates are applied. If left empty, the namespace defined in the ModuleTemplates is used. Resources cannot be created in the kyma-system namespace.
```

Comment:
```suggestion
      namespace-description: Namespace where ModuleTemplates are applied. If left empty, the namespace defined in the ModuleTemplates is used. Resources cannot be created in the `kyma-system` namespace.
```

Reply:
It doesn't work in yaml translations
<img width="534" height="105" alt="Screenshot 2025-11-26 at 08 38 05" src="https://github.com/user-attachments/assets/c250d4da-0558-4f9a-b787-9f095c9e8066" />

Reply:
ok, then, please ignore